### ISSUE TITLE
Gap fill - fill in the blanks. Hints are not available to screen readers

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1383]
The hints that appear in tooltips on the gap fill - fill in the blanks exercise (when mark at end is off) are not accessible to screen readers.
